scripts/install/install_tendermint_ubuntu.sh

```diff
@@ -22,12 +22,12 @@ curl -O https://storage.googleapis.com/golang/go1.10.linux-amd64.tar.gz
 tar -xvf go1.10.linux-amd64.tar.gz
 
 # move go binary and add to path
-mv go /usr/local
+sudo mv go /usr/local
```

Comment:
this should not require sudo.

Reply:
also, the above comment seems to be wrong. we're moving go folder, not binary to /usr/local, no?

Reply:
I'll address in a separate PR